test/e2e/fixtures.go

```diff
@@ -109,6 +110,16 @@ func skipIfHasWindowsNodes(tb testing.TB) {
 	}
 }
 
+func skipIfKubeApiServerFeatureGateNotEnable(tb testing.TB, featureGateName string) {
+	_, stdout, _, err := provider.RunCommandOnNode(clusterInfo.controlPlaneNodeName, "ps aux | grep apiserver | grep feature-gates")
```

Comment:
the function sounds generic but will work for features that are not enabled by default. If a newer K8s version enables it by default, it will skip a feature by mistake. It may not be a real problem in CI if we test few K8s versions only. But we should add a comment to explain its limitation.

Or for this particular case, you could create a Policy with endport set, check if the created policy still has it. If not, the feature is not enabled and the test can skip.

Reply:
Sounds good. Addressed your last suggestion.